source/solvers/heat_transfer.cc

```diff
@@ -454,8 +454,10 @@ HeatTransfer<dim>::assemble_local_system_matrix(
   const DoFHandler<dim> *dof_handler_fluid =
     multiphysics->get_dof_handler(PhysicsID::fluid_dynamics);
 
-  typename DoFHandler<dim>::active_cell_iterator velocity_cell(
-    &(*triangulation), cell->level(), cell->index(), dof_handler_fluid);
+  typename DoFHandler<dim>::active_cell_iterator fd_cell(&(*triangulation),
```

Comment:
Is fd meant to mean "fluid dynamics"? Maybe I would use a more meaningful name here.

Reply:
I don't mind the fd here to be honest. All the fluid dynamics functions have fd, heat transfers have ht, etc. Right now this is a standard nomenclature in the code, so I would not change that unless we change it everywhere and that owuld be another PR in itself.